team-validator.js

```diff
@@ -214,6 +215,14 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		if ((set.nature).id && !(set.nature).exists) {
```

Comment:
`set.nature` should be a string, not a Nature object. You still need to get the Nature here.